File: Functions/notes.c
// Anthony Petersen, C Functions Notes
#include <stdio.h>
int num;
//char name[50], place[50], verb[50];
//const char* word(char type[50]){
   // char choice[50];
    //print("Please give a %s", type);
    //scanf("%s", choice);
    //return choice;
    
//}
void due(char assignment[50], char day[20]){
    printf("The %s assignment is due %s", assignment, day);

}
int main(void){
    //printf("Please tell me a number: \n");
    //scanf("%d", num);
    //printf("%d", add(int numone+12));
    due("functions notes", "today\n");
    due("Hello world update", "tommorow\n");
    due("Financial calculator update", "Friday\n");
    return 0;
}

File: Functions/notes.py
#Anthony Petersen, Functions notes
#Functions hold actions to be reused
numone = int(input("Please enter a number: \n"))
numtwo = int(input("Please enter another number: \n"))
def add(numone, numtwo):
    print(numone**numtwo)
    return numone+numtwo
add(numone,numtwo)
def values(type):
    return input(f"Please enter a {type}: \n")
name = values("name").strip().capitalize()
place = values("place").strip().capitalize()
verb = values("past tense verb").strip().lower()
print(f"{name} was really hungry well going to {place} because they {verb} the whole way there")